csrc/scheduler/pointwise.cpp

```diff
@@ -682,17 +682,16 @@ void schedulePointwise(Fusion* fusion, const PointwiseParams* pparams) {
     // to do this is with Dependency check which will grab all intermediate
     // values too.
     auto lhs_all_vals = DependencyCheck::getAllValsBetween(
-        {reference_tv->getLogicalDomain().begin(),
-         reference_tv->getLogicalDomain().begin() + device_aware_break_point},
+        {ref_orig_loop.begin(),
```

Comment:
I guess this doesn't matter since we do `+ num_device_dims` at line 687, but IIUC, here we want to find all loop IDs that are derived from `ref_orig_loop[num_device_dims:device_aware_break_point]`, so, it seems to make more sense to do:

```suggestion
        {ref_orig_loop.begin() + num_device_dims,
```